packages/x-scheduler/src/joy/event-calendar/EventCalendar.types.ts

```diff
@@ -0,0 +1,12 @@
+import { CalendarEvent } from '../models/events';
+
+export interface EventCalendarProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The events to render in the calendar.
+   */
+  events: CalendarEvent[];
```

Comment:
Do you think we should treat it a a regular model (with a control prop and a default prop), or have a required control prop like the `row` on the grid?

I went with the grid's `row` approach because I guess we will have stuff like data-loaders at some point so it's easier to just follow the same approach.

Reply:
I think this approach seems good enough, at least as long as we don't see a clear shortcoming with it. 👌